src/Exceptions/HttpException.php

```diff
@@ -29,39 +30,17 @@ public static function buildHtmlResponse(): HtmlResponse
         $phrase = static::PHRASE;
         $href = static::HREF;
 
-        $html = <<<TEMPLATE
-        <!DOCTYPE HTML>
-        <html lang='en'>
-        <head>
-            <title>{$code} - {$phrase}</title>
-        </head>
-        <body>
-            <style>
-                * {
-                    font-family: Arial, Helvetica, sans-serif;
-                    text-align: center;
-                }
-
-                body {
-                    background: #1b1c1d;
-                    color: white;
-                    padding-top: calc(50vh - 95px);
-                }
-
-                body > div {
-                    max-width: 90%;
-                    margin: auto;
-                    width: fit-content;
-                }
-            </style>
-            <div>
-                <h1>{$code} - <a href='{$href}'>{$phrase}</a></h1>
-                <hr>
-                <p>{$description}</p>
-            </div>
-        </body>
-        </html>
-        TEMPLATE;
+        if (empty($phrase)) {
+            throw new RuntimeException("Phrase constant is not defined.");
+        }
+
+        if (!strlen($description)) {
+            throw new RuntimeException("Description constant defined.");
+        }
+
+        ob_start();
```

Comment:
**Comment**
Proper templating system where you can return a `view` would be nicer, but if you were gonna do that, it wouldn't be in this PR

Reply:
Didn't want to add unnecessary bloat to the package